docs/release-notes/release-2024-06-10.md

```diff
@@ -0,0 +1,258 @@
+---
+title: 0.57 - June 2024
+---
+
+:::warning
+This release contains deprecations
+:::
+
+## Values In TypeSpec
+
+Decorators often expect some sort of configuration to be passed as an argument. In version `0.51.0` we introduced the `valueof` keyword to constrain a decorator or template parameter to be a value, however this was limited to string, numeric and boolean values.
+When dealing with more complex decorators or free form data one had to use a model type to represent the data. This required a lot of manual validation in the decorator implementation and introduce a lot of potential issues(Passing a subtype like a union instead of a model)
```

Comment:
```suggestion
When dealing with more complex decorators or free form data, one had to use a model type to represent the data. Unfortunately, this approach was difficult for decorator authors to handle reliably and required somewhat sophisticated validation to avoid some common pitfalls that couldn't be prevented through type system constraints.
```